tests/configs/jobs.yaml

```diff
@@ -12,13 +12,15 @@ jobs:
     template: 'kbuild.jinja2'
     image: 'gcc-10:{arch}{fragments}'
     conditions: *checkout
+    kind: kbuild
     params:
       config: x86_64_defconfig
 
   kunit: &kunit-job
     template: 'kunit.jinja2'
     image: 'gcc-10:x86-kunit-kernelci'
     conditions: *checkout
+    kind: node
```

Comment:
Shouldn't the `kind` here be changed to `test` or `kunit`?

Reply:
Maybe at some point it should be something like `test` or something else, but probably not `kunit` as that's the test name.  For now there's only `node` as the generic kind and `kbuild` for kernel builds.  I'm working on adding LAVA tests now so we might have a new kind for that, depending on how things go with the `kbuild` thing and `Node.data` sub-schema.

Reply:
OK